internal/clients/idler.go

```diff
@@ -24,8 +29,13 @@ type Status struct {
 	IsIdle bool `json:"is_idle"`
 }
 
-//IsIdle returns true if Jenkins is idled for a given namespace
-func (i Idler) IsIdle(namespace string) (bool, error) {
+// IsIdle returns true if the Jenkins instance for the specified tenant is idled. False otherwise
+func (i Idler) IsIdle(tenant string) (bool, error) {
+	namespace := tenant
+	if !strings.HasSuffix(tenant, namespaceSuffix) {
+		namespace = tenant + namespaceSuffix
```

Comment:
This should really rely on the Tenant Service provided namespace name of type "jenkins" instead of hardcoding a naming pattern

Reply:
Can you create an issue and also point to the API to use?